src/bitcoin/bitcoind.rs

```diff
@@ -230,6 +230,37 @@ impl Client {
         Ok(())
     }
 
+    pub async fn derive_addresses(
```

Comment:
:question: Is this one just meant to be used for testing?

Reply:
We only use it in test BUT it is a valid command for testnet and mainnet bitcoind. This is why I don't put it behind a `#[cfg(test)]` compiler attribute.

The reason why I put the `generate_to_address` function behind `#[cfg(test)]` is because this command is only valid on regtest.